src/core/interaction/events/gestureevent.cpp

```diff
@@ -74,4 +75,12 @@ void GestureEvent::setScreenPosNormalized(dvec2 posNorm) { screenPosNorm_ = posN
 
 uint64_t GestureEvent::hash() const { return chash(); }
 
+void GestureEvent::print(std::ostream& ss) const {
+    util::printEvent(ss, "GestureEvent", std::make_pair("state", state_),
+                     std::make_pair("type", type_), std::make_pair("pos", screenPosNorm_),
+                     std::make_pair("depth", depth_), std::make_pair("canvasSize", canvasSize_),
+                     std::make_pair("deltaPos", deltaPos_),
+                     std::make_pair("modifiers", modifiers_));
+}
+
 }  // namespace inviwo
```

Comment:
missing newline

Reply:
All of our templates for code generation are missing newlines :) https://github.com/inviwo/inviwo/blob/master/tools/templates/

Reply:
those are deprecated. All the new ones in tools/meta/templates has newlines...